Arelle/Arelle
# Add Support For Saving Both OIM JSON and CSV Formats

#### Reason for change
We need the ability to save both JSON and CSV OIM formats in a single request.

#### Description of change
* Cleanup saveLoadableOIM plugin.
* Fix error handling bugs in saveLoadableOIM plugin.
* Add option to save all OIM formats to a directory (`--saveLoadableOIMDirectory`).

#### Steps to Test
* Run: `python arelleCmdLine.py --plugins saveLoadableOIM --file https://filings.xbrl.org/724500GJBUL3D9TW9Y18/2023-12-31/ESEF/NL/0/724500GJBUL3D9TW9Y18-2023-12-31-en.zip --saveLoadableOIMDirectory oim`
* Verify that both JSON and CSV OIM formats are saved into the `oim` directory.

**review**:
@Arelle/arelle
